MapboxNavigation/RouteTableViewController.swift

```diff
@@ -84,15 +84,7 @@ class RouteTableViewController: UIViewController {
             }
         }
     }
-    
-    func notifyDidChange(routeProgress: RouteProgress) {
-        showETA(routeProgress: routeProgress)
-    }
-    
-    func notifyDidReroute() {
-        tableView.reloadData()
```

Comment:
Don't we still need to refresh this after a reroute?

Reply:
Good catch